docs/qs-backup-setup.md

```diff
@@ -82,81 +89,155 @@ You should obtain a layout similar to this one (it is fine if you only have one
 	* **End on**: *Never*
   3. Press the **Create** button
 
+Your final BitShelter settings should look similar to the following configuration:
+
+<img src="/content/images/backup-setup/bitshelter-rules.png" data-origin="content/images/backup-setup/bitshelter-rules.png" alt="">
```

Comment:
No idea why this was originally in html format. Better to use markdown where applicable